notes/time_lock_notes.txt

```diff
@@ -0,0 +1,33 @@
+TIMELOCK NOTES
```

Comment:
Is this file gonna stay?

Reply:
I think this file should go once we implement time locks.

Reply:
Can you please add this note at the top of the file then resolve this comment?

Reply:
Done.